dashboard/app/models/script.rb

```diff
@@ -1328,6 +1328,12 @@ def summarize_i18n(include_lessons=true)
     data
   end
 
+  def summarize_i18n_for_display(include_lessons=true)
```

Comment:
I split `summarize_i18n` into `_for_display` and `_for_edit` to try to clarify why they were different. I am open to better naming. In other cases, `summarize` methods appear to generally be "for display", and we have used `summarize_for_edit` when we want to show things on the edit page, so I _could_ rename this back to `summarize_i18n` and follow that pattern, if folks think it's clear enough.

Reply:
I always found `summarize` to be too vague a name, so I like this change personally.